lib/SetupChecks/CirclesAppIsEnableCheck.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\Collectives\SetupChecks;
+
+use OCP\App\IAppManager;
+use OCP\IL10N;
+use OCP\SetupCheck\ISetupCheck;
+use OCP\SetupCheck\SetupResult;
+
+class CirclesAppIsEnableCheck implements ISetupCheck {
+	public function __construct(
+		private IL10N       $l10n,
+		private IAppManager $appManager
+	) {
+	}
+
+
+	public function getCategory(): string {
+		return 'app';
+	}
+
+	public function getName(): string {
+		return $this->l10n->t('Circles App Enabled');
+	}
+
+	public function run(): SetupResult {
+		if ($this->appManager->isEnabledForUser('circles')) {
+			return SetupResult::success($this->l10n->t('Circles App is enabled'));
```

Comment:
```suggestion
			return SetupResult::success($this->l10n->t('Teams app is enabled'));
```

We renamed the app to teams (sorry that wasn't reflected in the original ticket)